chall05/egarcia-.c

```diff
@@ -0,0 +1,30 @@
+#include <stdio.h>
+
+#define funtion(x)       printf(x" = %d\n", ft_ie_except_after_c(x));
+
+int     ft_ie_except_after_c(char* str)
+{
+    while (*str)
+    {
+        if (*(str + 2) && *str == 'c' && *(str + 1) == 'i' && *(str + 2) == 'e')
```

Comment:
Watch out... Here you will be reading after the end of the string, it will crash, and it will be your fault :(